### ISSUE TITLE
changing only some booleans in the example config and it returns a completely irational error (VIDEO)

### ISSUE THREAD

**Issue by Sladur23** [PokemonGoF/PokemonGo-Bot#5899]
Edit: Yep, I spelled irrational wrong... heat of the moment :)
Everything you need to see is in the video: https://vimeo.com/202545106

Password: bot123
 Too long didn't watch:
If i use the example config.json i get no error and the bot is working perfectly fine. But if i just change some falses to trues or vice versa i get an error telling me there is unexpected code in line 654 but there the config ends at line 653...: 
![asd](https://cloud.githubusercontent.com/assets/25365005/22620268/69b79298-eb08-11e6-8b22-f9028a6a7968.png)

**Comment by MerlionRock** [comment]
Sometimes is because you accidentally remove a "," or add a "," when there shouldn't be any.

Use pastebin.com to paste your config file and remove any sensitive data (if any) and show us.

**Comment by Sladur23** [comment]
Thanks for your reply!
I did not change any of the files, yet this error is gone and the bot logs in perfectly but i get another error now. (I swear to god I did not change anything :D)

New log / error message: http://pastebin.com/U8CLz7WT

config.json: http://pastebin.com/Gp7EMuJe

**Comment by MerlionRock** [comment]
@Sladur23 Latest commit is 7c2722d, your's is 7c2722d7. You are using Window? In your .bat, there should be a PokemonGo-Bot-Updater.bat. Run that file and try again.

**Comment by davidakachaos** [comment]
Master or dev branch?

**Comment by Sladur23** [comment]
Master :)

**Comment by Sladur23** [comment]
So I tried the new master release. Still the same issue. Just some line numbers changed:
http://pastebin.com/icyDbnNT

**Comment by Sladur23** [comment]
@MerlionRock 
@davidakachaos 

Now i did what was suggested here: https://github.com/PokemonGoF/PokemonGo-Bot/issues/2909
Also I am using the new master branch and running Kitematic in administrator mode. The error now has changed (I understand it is not an decoding error anymore):

`Traceback (most recent call last):
  File "pokecli.py", line 865, in <module>
    main()
  File "pokecli.py", line 206, in main
    bot.tick()
  File "/usr/src/app/pokemongo_bot/__init__.py", line 778, in tick
    if worker.work() == WorkerResult.RUNNING:
  File "/usr/src/app/pokemongo_bot/cell_workers/buddy_pokemon.py", line 135, in work
    if self._km_walked() - self.buddy['last_km_awarded'] >= self.buddy_distance_needed:
KeyError: 'last_km_awarded'`

**Comment by Sladur23** [comment]
Alright, fixed it:

What I did was:
1. I added these lines to pokecli.py:
`def main():
    try:
        reload(sys)
        sys.setdefaultencoding('utf8')
        logger.info('PokemonGO Bot v1.0')` Source: https://github.com/PokemonGoF/PokemonGo-Bot/issues/2909
2. I then build the docker image again, starting Kitematic in admin mode.
3. The above error was fixed by just moving as long in real life as i had one buddy reward. So this is the acutal workaround here and explains why I am the only one having this problem since verybody else probably got at least one of these rewards by now.
